libs/openFrameworks/app/ofWindowSettings.h

```diff
@@ -119,13 +119,14 @@ class ofGLWindowSettings: public ofWindowSettings{
 class ofGLESWindowSettings: public ofWindowSettings{
 public:
 	ofGLESWindowSettings()
-	:glesVersion(1){}
+	:glesVersion(1), preserveContextOnPause(true){}
```

Comment:
Just saw this. This class, ofGLESWindowSettings is used in other platforms and is a generic settings class for anything that uses GLES so this shouldn't be here.

If we want to add a new setting that is android specific it should go in an ofxAndroidWindowSettings class that extends this one.

We do something similar with ofGLWindowSettings and ofGLFWWindowSettings if you want to take a look

Reply:
Right, I looked at the *WindowSettings implementation and got scared by those wild casts in copy constructors :-). I'll change it.

Reply:
I know I should probably replace them with an abstract class that every settings of the same class implement instead of using inheritance but if you can just copy that style that should be ok by now

Reply:
Done. It is in e8a0d92799a22505b1e4aba45f81edfe44b2dfa4.